src/jobTree/jobStores/awsJobStore.py

```diff
@@ -76,7 +76,8 @@ def __init__( self, region, namePrefix, config=None ):
         self.jobs = self._getOrCreateDomain( 'jobs', create )
         self.versions = self._getOrCreateDomain( 'versions', create )
         self.files = self._getOrCreateBucket( 'files', create, versioning=True )
-        self.stats = self._getOrCreateBucket( 'stats', create )
+        self.stats = self._getOrCreateBucket( 'stats', create, versioning=True )
+        self.bucketMap = {'files':self.files, 'stats':self.stats}
```

Comment:
Having an additional dictionary is neither easy nor more flexible than using getattr. Please read this: https://docs.python.org/2/library/functions.html#getattr